### ISSUE TITLE
feature request: enhance retrieval of Compose identifiers

### ISSUE THREAD

**Issue by rulogarcillan** [appium/appium-inspector#900]
### Current Behavior

When using appium inspector with views created with Jetpack compose, accessibility and view semantics are shared in the same compose tree. To work with accessibility, sometimes you have to use the semantics where the "clearAndSetSemantics" tree is cleaned. This has a consequence, which is that all the child nodes are eliminated and merged into one, losing identifiers.

### Suggested Solution

Google in its documentation says to use the parameter ```useUnmergedTree = true``` to get the original tree without merging.

Can you make use of this parameter in the inspector so that we can get the tree ids without problems?

### Additional Information

https://developer.android.com/jetpack/compose/testing

**Comment by mykola-mokhnach** [comment]
AFAIK Espresso driver always applies useUnmergedTree argument value to `true` when showing the page source or looking for elements: https://github.com/appium/appium-espresso-driver/blob/75da960a27ca862cc140f31d53e6b4ef664162c2/espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SourceDocument.kt#L335

**Comment by rulogarcillan** [comment]
thanks, since which version, maybe it is not updated

**Comment by KazuCocoa** [comment]
No, UIA2 does not have the compose dependency as the architecture. UIA2 can interact with elements via AccessibilityNodeInfo. It looks like useUnmergedTree needs composeTestRule, then it is available in Espresso driver only.
https://github.com/appium/appium-espresso-driver/tree/master#jetpack-compose-support

**Comment by eglitise** [comment]
Closing as not an issue
